lib/network/test_util.go

```diff
@@ -0,0 +1,138 @@
+package sebaknetwork
```

Comment:
IMHO, tests.go is convention for this kind of utils

Reply:
ummm .. according to your comment , when I made some functions for test, I need to follow this convention. that is  you meant to ?

Reply:
I think that(test.go) is an implicit convention.

Reply:
Aha. I see. 
thanks I will divide to do that for test. 
And.. about `missing_transaction_test.go` , I also agree that if you think existing test code was meaningless. 
so you will tell me it is meaningless I will remove.